README.md

```diff
@@ -213,6 +213,31 @@ poetry shell
 ./deploy_doc.sh
 ```
 
+## Installing cardano-clusterlib in development mode
+
+Sometimes it is useful to test local changes made to [cardano-clusterlib](https://github.com/input-output-hk/cardano-clusterlib-py).
+To install cardano-clusterlib in development mode:
+
+```sh
+# activate virtual env
+poetry shell
```

Comment:
poetry shell shouldn't be called after the setup_env?

Reply:
No, you want to update your current activated venv, not create new venv from scratch.